tests/network/vmi_multus.go

```diff
@@ -761,6 +748,8 @@ var _ = Describe("[Serial]SRIOV", func() {
 		}
 
 		It("should block migration for SR-IOV VMI's when LiveMigration feature-gate is on but SRIOVLiveMigration is off", func() {
+			Expect(createNetworkAttachementDefinition(sriovnet1, util.NamespaceTestDefault, sriovConfCRD)).To((Succeed()), "should successfully create the network")
```

Comment:
I prefer for this change, just to place it all in the `BeforeEach` and not distribute it all over.
We need to re-organize the tests here, lets defer this part to that step.